pex/pex_builder.py

```diff
@@ -1,31 +1,33 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from __future__ import absolute_import, print_function
```

Comment:
Why remove this?

Reply:
There is no use of `print`.

Reply:
In Pants, we require all 4 future imports regardless of use, but it seems PEX does not do that. 

I advocate for always requiring the 4 future imports because it makes sure we never forget it when necessary. Print is fairly obvious and easy to remember; division is a much trickier one and easy to forget. 

It’s outside the scope of this PR to change how we handle future imports in PEX overall, so fine to remove print here, although I think there is more benefit than harm in keeping it.